### PR TITLE
Timeout of 25s is too small for "hanged" XHRs (close #347)

### LINKED ISSUE #347: Timeout of 25s is too small for `hanged` XHRs
The [Large Database Demo](https://demos.devexpress.com/ASPxGridViewDemos/DataBinding/LargeDatabase.aspx) uses a long-polling XHR to wait until a database is created on a server. It takes approx. 50s to create it, so hammerhead aborts the XHR. It leads to a test hanging, because demo page doesn't receive response from server.

### PR DESCRIPTION
**Pull request by churkin**
\сс @miherlosev @AndreyBelym

### CODE REVIEW

**Comment by miherlosev** [comment]
lgtm

**Comment by AndreyBelym** [comment]
lgtm

File: src/request-pipeline/destination-request/index.js [hunk 1]
```diff
@@ -19,9 +19,10 @@ function isDNSErr (err) {
 
 // DestinationRequest
 export default class DestinationRequest extends EventEmitter {
-    constructor (opts) {
+    constructor (opts, isXhr) {
```
Comment (inikulin) [inline src/request-pipeline/destination-request/index.js]: JUST WHY? You already have `isXhr` in `opts`.

**Comment by inikulin** [comment]
Please, fix this.

**Pull request merged**